include/fmt/core.h

```diff
@@ -1786,7 +1804,7 @@ class format_arg_store
         basic_format_args<Context>(*this),
 #endif
         data_{detail::make_arg<is_packed, Context>(args)...} {
-    if (num_named_args != 0)
+    if constexpr (num_named_args != 0)
```

Comment:
Using `const_check` is generally preferred for this warning avoidance.
https://github.com/fmtlib/fmt/blob/bd393456ed94a5d64bb6078aa3fa477ceeb621d9/include/fmt/core.h#L319-L320

Reply:
Thank you, done.